source/package.json

```diff
@@ -36,10 +37,10 @@
     "destroy": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region cdk destroy -v true -a \"node bin/deploy-client-stack.js\" && STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk destroy -v true",
     "update-es-logs-and-stack-variables": "bash ../deployment/custom-deployment/bin/update-es-logs-and-client-stack-vars.sh",
     "update-pdf-lambda-code": "bash ../deployment/custom-deployment/bin/update-pdf-lambda-code.sh",
-    "dev": "NEXT_PUBLIC_ENABLE_KENDRA=$npm_package_enableKendra next app",
-    "build": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region yarn pre-build && NEXT_PUBLIC_ENABLE_KENDRA=$npm_package_enableKendra next build app",
+    "dev": "NEXT_PUBLIC_ENABLE_KENDRA=$npm_package_enableKendra NEXT_PUBLIC_ENABLE_COMPREHEND_MEDICAL=$npm_package_enableComprehendMedical next app",
+    "build": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region yarn pre-build && NEXT_PUBLIC_ENABLE_KENDRA=$npm_package_enableKendra NEXT_PUBLIC_ENABLE_COMPREHEND_MEDICAL=$npm_package_enableComprehendMedical next build app",
     "export": "yarn build && next export app",
-    "pre-build": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region isROMode=$npm_package_isROMode ENABLE_KENDRA=$npm_package_enableKendra node ./bin/pre-build.js",
+    "pre-build": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region isROMode=$npm_package_isROMode ENABLE_KENDRA=$npm_package_enableKendra ENABLE_COMPREHEND_MEDICAL=$npm_package_enableComprehendMedical node ./bin/pre-build.js",
```

Comment:
Is there a reason to pass the ENV var to pre-build.js?

Reply:
nope. Will remove it